app/assets/javascripts/listings/templates/listing/_panel-apply-leasing-agent.html.slim

```diff
@@ -1,8 +1,9 @@
-p.content-group_address.c-steel
-  | {{ listing.Leasing_Agent_Street }}
-  br
-  | {{ listing.Leasing_Agent_City }} {{ listing.Leasing_Agent_State }} {{ listing.Leasing_Agent_Zip }}
-a.t-small.d-block.ui-link.margin-bottom.hide-for-print translate="LABEL.VIEW_MAP" title="{{ 'LABEL.VIEW_MAP' | translate }}" target="_blank" ng-href="https://www.google.com/maps/place/{{formattedLeasingAgentAddress(listing)}}"
+span ng-if="listing.Leasing_Agent_Street"
```

Comment:
don't use `span` to contain other block elements like this, in that case always use a `div`

Reply:
@kaptron updated :)